### PR TITLE
Update DEVELOPMENT.md: hot reloading instructions

### PR DESCRIPTION
**Pull request by coopy**
Depends on adding `builder run hot-open` to `builder-victory-component`

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -43,6 +43,20 @@ From there you can see:
...
+We use [`react-hot-loader`](https://github.com/gaearon/react-hot-loader#react-hot-loader-) to enable runtime-replacement of react components. This takes live reloading one step further by keeping the browser state intact and updating your component definition in-memory.
+
+```sh
+$ builder run hot       # hot reloading dev test/app server (OR)
+$ builder run open-hot  # hot reloading dev servers _and a browser window opens!_
+```
+
+From there you can see:
+
+* Demo app: [127.0.0.1:3000](http://127.0.0.1:3000/webpack-dev-server/)
```
Comment (ryan-roemer) [inline DEVELOPMENT.md:57]: `127.0.0.1:3000` should work... Can you point me to an easy repo to try this out in?

Comment (ryan-roemer) [inline DEVELOPMENT.md:57]: I'm updating this text snippet in `formidable-react-component-boilerplate` and `http://127.0.0.1:3000/` works straight up with hot reload (I'm editing and watching live DOM diffs right now!)

Comment (ryan-roemer) [inline DEVELOPMENT.md:57]: I have https://github.com/FormidableLabs/formidable-react-component-boilerplate/pull/58 up now -- `http://127.0.0.1:3000/` works as expected and I have a slightly different text snippet that you may wish to copy here.

**Comment by coopy** [comment]
Closed in favor of https://github.com/FormidableLabs/builder-victory-component/pull/25

**Pull request closed without merging**